extensions/wikibase/module/langs/translation-en.json

```
@@ -205,12 +204,20 @@
     "warnings-messages/multi-valued-property-is-required-for-existing-item/body": "This property is expected to have more than one statement on each item but it has single statement, for instance on {example_entity}. If the item already has statements with this property in {wikibase_name}, then this warning can be ignored.",
     "warnings-messages/new-item-requires-property-to-have-certain-values/title": "{property_entity} requires property {added_property_entity} to have certain values.",
     "warnings-messages/new-item-requires-property-to-have-certain-values/body": "Property {added_property_entity} doesn't have appropriate values such as on item {example_entity}.",
+    "warnings-messages/new-item-requires-property-to-have-certain-values-with-suggested-value/title": "{property_entity} requires property {added_property_entity} to have value {item_entity}.",
+    "warnings-messages/new-item-requires-property-to-have-certain-values-with-suggested-value/body": "Property {added_property_entity} must have value set as {item_entity}, such as on item {example_entity}.",
     "warnings-messages/new-item-requires-certain-other-statement/title": "{property_entity} requires statement with property {added_property_entity}.",
     "warnings-messages/new-item-requires-certain-other-statement/body": "This property is expected to have another statement with property {added_property_entity} such as on item {example_entity}.",
+    "warnings-messages/new-item-requires-certain-other-statement-with-suggested-value/title": "{property_entity} requires statement with property {added_property_entity} with value set as {item_entity}.",
+    "warnings-messages/new-item-requires-certain-other-statement-with-suggested-value/body": "This property is expected to have another statement with property {added_property_entity} with value set as {item_entity} such as on item {example_entity}.",
     "warnings-messages/existing-item-requires-property-to-have-certain-values/title": "{property_entity} requires property {added_property_entity} to have certain values.",
     "warnings-messages/existing-item-requires-property-to-have-certain-values/body": "Property {added_property_entity} doesn't have appropriate values such as on item {example_entity}. If the item already has statements with property {added_property_entity} and with appropriate values in {wikibase_name}, then this warning can be ignored.",
+    "warnings-messages/existing-item-requires-property-to-have-certain-values-with-suggested-value/title": "{property_entity} requires property {added_property_entity} with value set as {item_entity}.",
+    "warnings-messages/existing-item-requires-property-to-have-certain-values-with-suggested-value/body": "Property {added_property_entity} must have value set as {item_entity} such as on item {example_entity}. If the item already has statements with property {added_property_entity} and with appropriate values in {wikibase_name}, then this warning can be ignored.",
```

I think the current wording is a bit ambiguous because it makes me think that as a user I should imitate whatever there is on `example_entity`, but it's the other way around: the `example_entity` is an example of where the statement is missing. So I would propose to improve the message as follows:
```suggestion
    "warnings-messages/existing-item-requires-property-to-have-certain-values-with-suggested-value/body": "On entities such as {example_entity}, property {added_property_entity} must have a value set to {item_entity}. If the entity already has statements with property {added_property_entity} and with appropriate values in {wikibase_name}, then this warning can be ignored.",
```